internal/impl/tmpl/devcontainerDockerfile.tmpl

```diff
@@ -1,30 +1,30 @@
-FROM alpine:3
+FROM debian:stable-slim
 
-# Setting up devbox user
-ENV DEVBOX_USER=devbox
-RUN adduser -h /home/$DEVBOX_USER -D -s /bin/bash $DEVBOX_USER
-RUN addgroup sudo
-RUN addgroup $DEVBOX_USER sudo
-RUN echo " $DEVBOX_USER      ALL=(ALL:ALL) NOPASSWD: ALL" >> /etc/sudoers
+# Step 1: Installing dependencies
+RUN apt-get update
+RUN apt-get -y install bash binutils git xz-utils wget sudo
```

Comment:
How about choose a base image that contains these tools (or some of them)?

Reply:
@hezhizhen The motivation for using this base image is that, we use the same base image in devbox cloud instances. To keep the image size small, it's better to use a minimal image and install the tools you need.

Reply:
I see.